class/teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super
+    @specialization = specialization
+  end
```

Comment:
- [ ]  Kindly make sure to call `super` with Required Arguments. In the `Teacher` class constructor, you're calling `super` without passing the required arguments that the parent class `Person` expects. Make sure to pass the `age` and `name` arguments to the `super` call in the `initialize` method of the `Teacher` class, so the parent class can initialize properly.